ingestion-beam/src/test/java/com/mozilla/telemetry/integration/BigQueryIntegrationTest.java

```diff
@@ -111,7 +111,8 @@ public void canWriteToBigQuery() throws Exception {
     String output = String.format("%s:%s", projectId, tableSpec);
 
     PipelineResult result = Sink.run(new String[] { "--inputFileFormat=json", "--inputType=file",
-        "--input=" + input, "--outputType=bigquery", "--bqWriteMethod=streaming",
+        "--input=" + input, "--outputType=bigquery", "--bqWriteMethod=file_loads",
+        "--tempLocation=gs://gcp-ingestion-static-test-bucket/temp/bq-loads",
```

Comment:
we should make this bucket a static constant that we re-use, but I can file that as a separate PR